### PR TITLE
Small Fixes

### PR DESCRIPTION
**Pull request by Mango-3**
- Correct Filter Size lower limit
- Rename Max Global Bounces to Max Ray Bounces

### CODE REVIEW

File: src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp [hunk 1]
```diff
@@ -1006,7 +1006,7 @@ ParamBlockDesc2 g_param_block_desc(
         p_accessor, &g_pblock_accessor,
     p_end,
 
-    ParamIdGlobalBounceLimit, L"global_illumination_bounces", TYPE_INT, P_TRANSIENT, 0,
+    ParamIdGlobalBounceLimit, L"max_ray_bounces", TYPE_INT, P_TRANSIENT, 0,
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp]: Are we breaking any form of backward compatibility with this change?

File: src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.rc [hunk 2]
```diff
@@ -116,7 +116,7 @@ IDD_FORMVIEW_RENDERERPARAMS_PATH_TRACING DIALOGEX 0, 0, 200, 210
 STYLE DS_SETFONT | WS_CHILD | WS_VISIBLE
 FONT 8, "MS Sans Serif", 0, 0, 0x1
 BEGIN
-    LTEXT           "Max Global Bounces:",IDC_STATIC_BOUNCES,0,35,68,8
+    LTEXT           "Max Ray Bounces:",IDC_STATIC_BOUNCES,0,35,61,8
     CONTROL         "Max Global Bounces",IDC_TEXT_BOUNCES,"CustEdit",WS_TABSTOP,105,34,16,10
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.rc]: Rename other associated controls.

File: src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp (hunk 1 above)
Comment (Mango-3) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp]: I haven't had issues loading older scenes but I'm not 100% sure. I will revert it back just to be on the safe side.

File: src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.rc (hunk 2 above)
Comment (Mango-3) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.rc]: ok

**Comment by dictoon** [comment]
Thanks! Could you combine the last two commits please?

**Pull request merged**